src/Connectors/GithubConnector.php

```diff
@@ -62,16 +65,29 @@ public function deploy(array $cmds = []) {
 		}
 		return $result;
 	}
-	
-	public function addAcceptedEvent($event) {
+
+	public function addAcceptedEvent($event)
+	{
 		$this->acceptedEvents[] = $event;
 	}
-	
-	public function verify(){
+
+	public function verify()
+	{
 		if ($this->repoSecret == null) return true;
 		// Split signature into algorithm and hash
 		list ($algo, $hash) = explode('=', $this->signature, 2);
 		$payloadHash = hash_hmac($algo, $this->payload, $this->repoSecret);
 		return $hash === $payloadHash;
 	}
+
+	public function getHeaders()
+	{
+		$headers = [];
```

Comment:
@sergey-i-che I'd add code
```
if (function_exists('getallheaders')) {
    return getallheaders();
}
```
before the line.

Reply:
@f0rtis for some reason locally I have apache_request_headers working and getallheaders alias not. so I added both functions checks